libs/components/lists/src/lib/modules/sort/sort.component.ts

```diff
@@ -18,6 +24,7 @@ export class SkySortComponent {
    * sort button's `aria-label` attribute to provide a text equivalent for screen readers
```

Comment:
Just like with filter, this property isn't showing up in the docs: https://developer.blackbaud.com/skyux/components/sort?docs-active-tab=development. Any idea what's going on here? I don't see what's causing this.

Reply:
Same answer - it is pending